frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayLicensesEdit.java

```diff
@@ -82,12 +69,11 @@ private void printEmptyLicenses(StringBuilder display) {
                 .append(String.format("<input class=\"clickable\" type=\"text\" readonly=\"\" placeholder=\"Click to set Licenses\" id=\"%sDisplay\" onclick=\"%s\" />", id, onclick));
     }
 
-    private void printLicenses(StringBuilder display, Collection<License> licenses) {
+    private void printLicenses(StringBuilder display, Collection<String> licenseIds) {
         Joiner commaJoiner = Joiner.on(", ");
-        Function<License, String> nameExtract = extractField(License._Fields.FULLNAME, String.class);
 
-        String licenseIdsStr = commaJoiner.join(transform(licenses, extractId()));
-        String licenseNamesStr = commaJoiner.join(transform(licenses, nameExtract));
+        String licenseIdsStr = commaJoiner.join(licenseIds);
+        String licenseNamesStr = commaJoiner.join(licenseIds);
```

Comment:
It's bound to be the same as `licenseIdsStr` (see previous line). Can be deleted.